src/pwncore/models/ctf.py

```diff
@@ -19,15 +23,19 @@ class Problem(Model):
     author = fields.TextField()
 
     image_name = fields.TextField()
-    image_config: Field[dict[str, list]] = fields.JSONField()  # type: ignore[assignment]
+    image_config: fields.Field[dict[str, list]] = fields.JSONField()  # type: ignore[assignment]
 
     hints: fields.ReverseRelation[Hint]
 
+    class PydanticMeta:
+        exclude = ["image_name", "image_config"]
+
 
 class Hint(Model):
+    id = fields.IntField(pk=True)
     order = fields.SmallIntField()  # 0, 1, 2
     problem: fields.ForeignKeyRelation[Problem] = fields.ForeignKeyField(
-        "models.Problem", related_name="hints"
+        "models.Problem"
```

Comment:
Readd the `related_name`

Reply:
We are not using it right now. Should I still readd it?

Reply:
I remember some code using hints to prefetch, if it was in this PR only then get rid of its annotation in Problem

Reply:
All prefetch queries were replaced with automatic accessing using Model.field syntax.

Reply:
then just remove the ReverseRelation annotation in the Problem model